### ISSUE TITLE
Invalid JSON payload

### ISSUE THREAD

**Issue by coffeverton** [jordikroon/Php-Google-Vision-Api#24]
Hi, i'm getting this message:

Invalid JSON payload received. Unknown name "image_context" at 'requests[0]': Proto field is not repeating, cannot start list.

Here's my code:

```php
<?php
$loader = require 'vendor/autoload.php';
$apiKey = 'myapikey';
$vision = new \Vision\Vision(
    $apiKey,
    [
        new \Vision\Feature(\Vision\Feature::TEXT_DETECTION, 100),
    ]
);

$imagePath = __DIR__.'/1167.jpg';

$response = $vision->request(
    new \Vision\Request\Image\LocalImage($imagePath)
);
var_dump($response);
```

The image is a small jpeg file.

Can you help me?
Thanks!

**Comment by jordikroon** [comment]
Hi @coffeverton. Thanks for reporting.

The API request to Google Vision throws an error when sending an empty ImageContext object (or null). I wasn't aware of this.

I just pushed a fix to 1.7.2. Let me know if you have any other issues after a `composer update`.

**Comment by coffeverton** [comment]
@jordikroon Now it works! Thank you!
